server/src/controllers/api/projects_controller.js

```diff
@@ -117,11 +118,22 @@ async function getProjects(req, res) {
       $sort: sortParams,
     },
     {
-      $limit: pageSize,
+      $facet: {
+        metaData: [
```

Comment:
Since we are getting the total count from the call to `/projects/counts` then we don't need this and we can remove it